### PR TITLE
refactor: enable build_graph to be invoked from the service

### PR DESCRIPTION
**Pull request by jsam**

### CODE REVIEW

**Review by mohammad-sdsc — APPROVED** [review]
LGTM, thanks!

File: renku/cli/log.py [hunk 1]
```diff
@@ -96,24 +96,7 @@
...
-        commit = client.repo.rev_parse(stop)
-        paths = (
-            str(client.path / item.a_path)
-            for item in commit.diff(commit.parents or NULL_TREE)
-            # if not item.deleted_file
-        )
-
-    # NOTE shall we warn when "not no_output and not paths"?
-    graph.build(paths=paths, revision=revision, can_be_cwl=no_output)
+    graph = build_graph(revision, format, no_output, paths)
```
Comment (mohammad-sdsc) [inline renku/cli/log.py:101]: `format` should not be here

Comment (jsam) [reply renku/cli/log.py]: OH :see_no_evil: 

tyvm! :taco: :tada:

File: renku/core/commands/graph.py [hunk 2]
```diff
@@ -510,3 +512,26 @@ def workflow_has_identical_subprocesses(workflow_, subprocesses_ids_):
                     return workflow.association.plan
 
         return run
+
+
+@pass_local_client(requires_migration=True)
+def build_graph(client, revision, no_output, paths):
+    """"""
```
Comment (mohammad-sdsc) [inline renku/core/commands/graph.py:519]: It's still complaining 😄 : `D103: Missing docstring in public function`

Comment (jsam) [reply renku/core/commands/graph.py]: Yea I just noticed it in the tests. Pushed the fix right now :see_no_evil:

**Review by mohammad-sdsc — APPROVED** [review]
🚀

**Pull request merged**